src/sentry/overwatch_webhooks/overwatch_consent/impl.py

```diff
@@ -0,0 +1,44 @@
+# Please do not use
+#     from __future__ import annotations
+# in modules such as this one where hybrid cloud data models or service classes are
+# defined, because we want to reflect on type annotations and avoid forward references.
+
+from sentry.constants import ENABLE_PR_REVIEW_TEST_GENERATION_DEFAULT, HIDE_AI_FEATURES_DEFAULT
+from sentry.models.organization import Organization
+from sentry.overwatch_webhooks.overwatch_consent.model import RpcOrganizationConsentStatus
+from sentry.overwatch_webhooks.overwatch_consent.service import OverwatchConsentService
+
+
+class DatabaseBackedOverwatchConsentService(OverwatchConsentService):
+    def get_organization_prevent_consent_status(
```

Comment:
Is this for prevent or overwatch? Both are being used.

Reply:
Good question. I think it's mainly for overwatch (@giovanni-guidini correct me if I'm wrong on this please). I can rename this method.